paddle/infrt/dialect/tensorrt/trt_ops.cc

```diff
@@ -19,8 +19,7 @@
 #include "mlir/Interfaces/CallInterfaces.h"
 #include "mlir/Interfaces/SideEffectInterfaces.h"
 
-namespace infrt {
-namespace trt {
+namespace infrt::trt {
```

Comment:
infrt::trt is not supported in C++ 11 standard.
Please remain the original style.

Reply:
Done